CRM/Report/Form/Contact/Relationship.php

```diff
@@ -528,13 +561,12 @@ public function where() {
         }
       }
     }
-
-    if (empty($whereClauses)) {
-      $this->_where = 'WHERE ( 1 ) ';
-      $this->_having = '';
+    $this->_where = "WHERE ( {$this->_aliases['civicrm_contact']}.is_deleted = 0 AND {$this->_aliases['civicrm_contact_b']}.is_deleted = 0 ) ";
```

Comment:
I just ran into the is_deleted issue (trashed contacts were displaying in the report) and was going to file a PR, but see this already addresses it. The assumption throughout Civi is that trashed contacts are excluded unless options are explicitly provided to display them. I think the same should hold true here. Either add a filter option to include/not include trashed contacts, or hard code their exclusion. 

Personally, I think that although a filter option wouldn't hurt, it isn't necessary unless there's specific demand.